### ISSUE TITLE
Passing a startup script as metadata

### ISSUE THREAD

**Issue by dmytro** [terraform-google-modules/terraform-google-bastion-host#177]
### TL;DR

When passing a metadata block with startup script to the instance other metadata don't work.

### Expected behavior

According to [the documentation](https://github.com/terraform-google-modules/terraform-google-bastion-host/blob/3c2a66fcc392ba6319f0c8026d63024cacafc6b9/metadata.yaml#L143-L147) additional metadata should be added.  

### Observed behavior

1) Below is a working terraform code WITHOUT startup script. We are using roles/user groups to authorize login to the host. 

```
module "iap_bastion" {
 source    = "terraform-google-modules/bastion-host/google"
 name     = "bastion-host"
 project    = var.project_id
 machine_type = "e2-medium"
 zone     = "${var.region}-a"
 network    = module.vpc.network_name
 subnet    = module.vpc.subnets_names[0]
 image     = "debian-11-bullseye-v20210817"
 image_family = "debian-11"
 image_project = "debian-cloud"
 scopes    = ["cloud-platform"]
 version    = "5.3.0"
 members = [
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
 ]
 
  random_role_id = false
}
```

2) Following terraform code below is example of NOT WORKING code. The only difference is added `metadata` block with `startup-script` string.

```
module "iap_bastion" {
 source    = "terraform-google-modules/bastion-host/google"
 name     = "bastion-host"
 project    = var.project_id
 machine_type = "e2-medium"
 zone     = "${var.region}-a"
 network    = module.vpc.network_name
 subnet    = module.vpc.subnets_names[0]
 image     = "debian-11-bullseye-v20210817"
 image_family = "debian-11"
 image_project = "debian-cloud"
 scopes    = ["cloud-platform"]
 version    = "5.3.0"
 members = [
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
 ]

 random_role_id = false

 metadata = {
  startup-script = <<-EOF
#!/bin/bash
apt update
apt -y upgrade
apt -y install postgresql-client
apt -y install redis-server stunnel4 telnet
EOF
 }
}
```

### Problem:

* With added `metadata` block nobody is able login to the bastion-host;
* From IAP UI I can see that:
  * on the host with startup script roles are missing (see attached file no_groups.jpg), 
  * while  host without metadata block, has roles/user groups configured.
![no_groups](https://github.com/terraform-google-modules/terraform-google-bastion-host/assets/81929/3f5d2d33-8ed3-458b-9833-e3c37ac87449)

### Terraform Configuration

```hcl
module "iap_bastion" {
 source    = "terraform-google-modules/bastion-host/google"
 name     = "bastion-host"
 project    = var.project_id
 machine_type = "e2-medium"
 zone     = "${var.region}-a"
 network    = module.vpc.network_name
 subnet    = module.vpc.subnets_names[0]
 image     = "debian-11-bullseye-v20210817"
 image_family = "debian-11"
 image_project = "debian-cloud"
 scopes    = ["cloud-platform"]
 version    = "5.3.0"
 members = [
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
  "group:<redacted>@<redacted>",
 ]

 random_role_id = false

 metadata = {
  startup-script = <<-EOF
#!/bin/bash
apt update
apt -y upgrade
apt -y install postgresql-client
apt -y install redis-server stunnel4 telnet
EOF
 }
}
```
```


### Terraform Version

```sh
Terraform v1.6.6
```

**Comment by dmytro** [comment]
I've found, that I should use `startup_script` argument. Unfortunately it's not documented on the https://registry.terraform.io/modules/terraform-google-modules/bastion-host/google/latest page.
